containers/release-candidate/setup-container.sh

```diff
@@ -34,7 +34,7 @@ rm wix311-binaries.zip
 
 # enable sbt pgp
 mkdir -p /root/.sbt/1.0/plugins/
-sh -c "echo 'addSbtPlugin(\"com.jsuereth\" % \"sbt-pgp\" % \"1.1.1\")' >> /root/.sbt/1.0/plugins/pgp.sbt"
+sh -c "echo 'addSbtPlugin(\"com.jsuereth\" % \"sbt-pgp\" % \"2.0.0\")' >> /root/.sbt/1.0/plugins/pgp.sbt"
```

Comment:
Looks like the latest version is actualy 2.0.1, sbt-pgp just hasn't updated their README. Probably worth updating to that version.